args: -r -u -N -- newfile-1.0.14_orig/doc/Makefile.am newfile-1.0.14/doc/Makefile.am
--- newfile-1.0.14_orig/doc/Makefile.am	2008-06-24 01:05:20.000000000 +0200
+++ newfile-1.0.14/doc/Makefile.am	2008-06-24 01:14:18.000000000 +0200
@@ -41,7 +41,7 @@
 man1dir=	$(prefix)/man/man1
 
 rdoccpio=	rdoc.cpio.gz
-man1_DATA=	newfile.1
+man1_MAN=	newfile.1
 pkgdoc_DATA=	newfile.html
 
 install-data-hook: $(rdoccpio)
